### PR TITLE
Death messages for non-robotic station species no longer show up after two tiles of distance.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Opening for feedback. I expect this to be pretty controversial.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!wip

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/mob/death.dm [hunk 1]
```diff
@@ -48,15 +48,15 @@
...
+/mob/proc/death(gibbed,deathmessage="seizes up and falls limp...", messagerange)
 
 	if(stat == DEAD)
 		return 0
 
 	facing_dir = null
 
 	if(!gibbed && deathmessage != "no message") // This is gross, but reliable. Only brains use it.
-		src.visible_message("<b>\The [src.name]</b> [deathmessage]")
+		src.visible_message("<b>\The [src.name]</b> [deathmessage]", null, null, messagerange)
```
Comment (skull132) [inline code/modules/mob/death.dm]:
```suggestion
		src.visible_message("<b>\The [src.name]</b> [deathmessage]", range = messagerange)
```

File: code/modules/mob/death.dm [hunk 2]
```diff
@@ -48,15 +48,15 @@
 	QDEL_IN(animation, 15)
 	QDEL_IN(src, 15)
 
-/mob/proc/death(gibbed,deathmessage="seizes up and falls limp...")
+/mob/proc/death(gibbed,deathmessage="seizes up and falls limp...", messagerange)
```
Comment (skull132) [inline code/modules/mob/death.dm]:
```suggestion
/mob/proc/death(gibbed,deathmessage="seizes up and falls limp...", messagerange = world.view)
```

**Pull request merged**